CMakeLists.txt

```diff
@@ -124,8 +114,7 @@ if(WITH_DAGMC)
       set(DAGMC_ROOT "${DEPS_ROOT_DIR}")
     endif()
     MESSAGE("-- DAGMC Root: ${DAGMC_ROOT}")
-    find_package(DAGMC)
-    message("-- DAGMC Found: ${DAGMC_FOUND}")
```

Comment:
Some sort of message about what is found for DAGMC would be useful

Reply:
I noticed that, FindDAGMC comes with some logs. However, let me know if you still want to add the logs here as well.

Reply:
I think we just want a message about what version of DAGMC is found

Reply:
Updating FindDAGMC file.